test/integration/osl_mysql_test/controls/osl_mysql_test.rb

```diff
@@ -1,6 +1,6 @@
 # Ensure that the port is bound, and the service is running.
 
-describe service('mysql.service') do
+describe service('mariadb.service') do
```

Comment:
This needs to be different based on which OS/Version you're on. Match it to what it needs to be on each platform please.

Reply:
All versions of mariadb use the systemd unit `mariadb`. But, for some reason, later versions also mimic the `mysql` unit. Changing it to check for `mariadb` allows to easily check all version.